### ISSUE TITLE
TGCS Workshop Planning

### ISSUE THREAD

**Issue by nickbianco** [opensim-org/opensim-core#2982]
[TGCS Workshop Proposal](https://docs.google.com/document/d/17MCMD_IdWBSkrqvS6EXyb3DcP_IUNf_DkGs67ufGE-I/edit)

### IMU-tracking problem
- [x] Add options to MocoAccelerationTrackingGoal for gravity offset and converting accelerations to the IMU frame (#2961)
- [ ] Create simulation utility for creating synthetic IMU signals
- [ ] Squat-to-stand "round trip" follow-along Matlab and Python examples (e.g., exampleSquatToStandIMUTracking, exampleIMUTracking)

### EMG-driven problem
- [ ] Add option for scale factor optimization to Moco (#2903)
- [ ] Create standalone Matlab and Python examples based on exampleMocoInverse for EMG-driven simulation in follow-along format (e.g., exampleEMGDriven)

### Python examples
- [ ] Test [existing Python environment installation materials](https://simtk-confluence.stanford.edu/display/OpenSim/Scripting+in+Python) with novice Python users in the lab
- [ ] Provide office hours before meeting for Python setup

### Presentation materials
- [ ] Draft PowerPoint slides
- [ ] Draft email for pre-workshop installation instructions and info about office hours

**Comment by nickbianco** [comment]
@aymanhab @carmichaelong @antoinefalisse @jenhicks

**Comment by aymanhab** [comment]
Opened issue #2983 to discuss details of creating synthetic IMU signals

**Comment by aymanhab** [comment]
@nickbianco With IMU and IMUDataReporter now in place, I'm looking into GUI needs/functionality for the demo, is there an outline or documentation that I should check? Also do you assume/use a noise model? Thanks

**Comment by aymanhab** [comment]
@nickbianco Do we need update doxygen for TGCS build? Also do we need a link from the advanced examples page on confluence to the new Moco Examples?

**Comment by nickbianco** [comment]
Yes, we will need to update Doxygen and the examples page for the new Moco examples.
